### DISCUSSION TITLE
Release Roadmap

### DISCUSSION THREAD

**Discussion by mpscholten** [digitallyinduced/ihp#604] [General]
Here's a short timeline for upcoming IHP releases.

- 08.01.2021: `v0.8.0` (we'll change the versioning format with begin of the new year)
- 26.02.2021: Release Candidate, `v0.9.0`
- 19.03.2021: Release Candidate, `v0.9.x`
    - Should include: https://github.com/digitallyinduced/ihp/issues/562 https://github.com/digitallyinduced/ihp/issues/506  
- 09.04.2021: We'll drop the Beta from IHP and tag IHP `v1.0.0`

**Comment by OhCoder** [comment]
nice.
